packages/renderer/src/lib/troubleshooting/TroubleshootingGatherLogs.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+import { faFileLines, faScroll } from '@fortawesome/free-solid-svg-icons';
+import Button from '../ui/Button.svelte';
+import Fa from 'svelte-fa';
+import { generateLogFileName } from '../../../../main/src/plugin/troubleshooting';
```

Comment:
I think here it's an issue as we import a class that is defined on the 'main' side and at some point it could not be anymore compliant with the client side

so probably need to be in a separate file